src/js/control-bar/play-toggle.js

```diff
@@ -68,6 +70,18 @@ class PlayToggle extends Button {
     this.controlText('Play');
   }
 
+  /**
+   * Add the vjs-ended class to the element so it can change appearance
+   *
+   * @method handleEnded
+   */
+  handleEnded() {
+    this.removeClass('vjs-playing');
+    this.addClass('vjs-ended');
+    // change the button text to "Replay"
+    this.controlText('Replay');
```

Comment:
"Replay" isn't in our translations so we should make sure it's in `en.json` and run `grunt check-translations` to make sure it gets added to the needed translations document.